build.gradle

```diff
@@ -466,20 +466,20 @@ def connectPkgs = ['connect:api', 'connect:runtime', 'connect:transforms', 'conn
 def pkgs = ['clients', 'examples', 'log4j-appender', 'tools', 'streams', 'streams:examples'] + connectPkgs
 
 tasks.create(name: "jarConnect", dependsOn: connectPkgs.collect { it + ":jar" }) {}
-tasks.create(name: "jarAll", dependsOn: ['jar_core_2_10', 'jar_core_2_11'] + pkgs.collect { it + ":jar" }) { }
+tasks.create(name: "jarAll", dependsOn: ['jar_core_2_11'] + pkgs.collect { it + ":jar" }) { }
```

Comment:
Would it be worth refactoring all these to map a centralized list of the release versions? i.e. so we can centralize the lists `release_versions = [ '2_11' ]` and `available_versions = [ '2_11', '2_12' ]` in one place? It'll reduce the number of places we need to update these values in the future (which I assume we'll do when we finally drop JDK8 and bring `2_12` into the officially released versions).